scss/components/_chart-area.scss

```diff
@@ -85,6 +85,11 @@
 
         p {
             margin: 0;
+            color: $white;
```

Comment:
Should we set this as the default text colour when the background colour is used as per the old way of managing colours?

Reply:
I don't know! Sounds like a question for Ben?

Reply:
Ah, nevermind, I see what you're talking about now.